src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -39,6 +38,12 @@ public static class Motors {
 		}
 	}
 
+	public static class Ultrasonic {
```

Comment:
Just a thought - maybe its time for multiple consts files?

Reply:
Raz - I agree, it's a good idea, but on a sperate branch